CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ Currently the versioning policy of this project follows [Semantic Versioning v2.
 ### Fixed
 - Fixed False positives for `RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE` on try-with-resources with interface references ([#1931](https://github.com/spotbugs/spotbugs/issues/1931))
 
+### Added
+* New bug type `CWO_CLOSED_WITHOUT_OPENED` for locks that might be released without even being acquired. (See[SEI CERT rule LCK08-J](https://wiki.sei.cmu.edu/confluence/display/java/LCK08-J.+Ensure+actively+held+locks+are+released+on+exceptional+conditions))
+
```

Comment:
Please merge the latest `master` branch in the upstream repo, then move this entry to under the `## Unreleased`.

It is also suggested to add a link to the related PR or issue, like `([#2055](https://github.com/spotbugs/spotbugs/pull/2055))`, at the end of this CHANGELOG entry.